src/widget/form/chatform.cpp

```diff
@@ -185,8 +185,12 @@ void ChatForm::onTextEditChanged()
 
 void ChatForm::onAttachClicked()
 {
-    QStringList paths = QFileDialog::getOpenFileNames(this,
-                                                      tr("Send a file"));
+    QStringList paths = QFileDialog::getOpenFileNames ( this,
```

Comment:
I saw this style used in a few other places thus I thought it was the accepted style, but I agree it doesn't looks very nice, and will change each occurrence to omit the space.